std_msgs/Header header

uint8 current_state
